spec/mailers/reservation_mailer_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+describe ReservationMailer, :type => :mailer do
+  it "it creates a notification for the host" do
+    user = create(:user)
+    ReservationMailer.notify(user, user).deliver_now
+    result = ActionMailer::Base.deliveries.last
+
+    expect result
+    expect result.to.include? ('kitpearosn@me.com')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.